docs/user_guide/assessment.rst

```diff
@@ -1,20 +1,144 @@
 Assessment
 ==========
 
-Metrics
--------
+Introduction
+------------
+
+The goal of fairness assessment is to answer the question: Which groups of 
+people may be disproportionately negatively impacted by an AI system and in 
+what ways?
+
+The steps of the assesment are as follows:
+1. Identify types of harms
+2. Identify the groups that might be harmed
+3. Quantify harms
+4. Compare quantified harms across the groups
+
+We next examine these four steps in more detail.
+
+Identify types of harms
+^^^^^^^^^^^^^^^^^^^^^^^
+
+See :ref:`types_of_harms` for a guide to types of fairness-related harms. 
+For example, in a system for screening job applications, qualified candidates 
+that are automatically rejected experience an allocation harm. In a 
+speech-to-text transcription system, disparities in word error rates for 
+different groups may result in harms due to differences in the quality of service.
+Note that one system can lead to multiple harms, and different types of 
+harms are not mutually exclusive. For more information, review 
+Fairlearn's `2021 SciPy tutorial <https://github.com/fairlearn/talks/blob/main/2021_scipy_tutorial/overview.pdf>`_.
+
+Identify the groups that might be harmed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In most applications, we consider demographic groups including historically 
+marginalized groups (e.g., based on gender, race, ethnicity). We should also 
+consider groups that are relevant to a particular use case or deployment context. For example, for 
+speech-to-text transcription, this might include groups who speak a regional dialect or people who are a  
+native or a non-native speaker.
+
+It is also important to consider group intersections, for example, in addition
+to considering groups according to gender and groups according to race, it is 
+also important to consider their intersections (e.g., Black women, Latinx 
+nonbinary people, etc.). Kimberlé Crenshaw's `work on intersectionality <https://www.jstor.org/stable/1229039>`_ 
```

Comment:
I believe this is the only edit we still require. Could you add a screenshot of the final page @alliesaizan?

```suggestion
nonbinary people, etc.). Kimberlé Crenshaw's work on intersectionality :footcite:p:`crenshaw1991intersectionality`
```

Reply:
The final HTML page? I am having some issues building the documentation on my end so that may be tricky.

Reply:
Any eay we can help eith thr issues?

Reply:
I'll try to troubleshoot it further this weekend. I primarily work on a Windows machine but can also try it on a Mac. and see if the HTML pages render. When I run the Windows [commands on the website](https://fairlearn.org/v0.7.0/contributor_guide/contributing_documentation.html), under docs>_build> the html folder is created, but its only content is a .doctrees folder. I'm guessing the issue is with sphinx itself, but I can give more details after I have looked into it a bit more.

Reply:
I am also using Windows. I'd try deleting the entire build directory so that it can start afresh.

Reply:
I was able to figure it out. I updated my Python version and was able to build the docs (some of the plots weren't rendering in my old Python environment, which was holding up the build.) Images of the first few sections are below.

![image](https://user-images.githubusercontent.com/15962391/162581534-30c810b5-3383-4df4-8923-f2aae811d598.png)

![image](https://user-images.githubusercontent.com/15962391/162581553-4e4ad48a-d899-48a8-8407-c62a804bef52.png)

![image](https://user-images.githubusercontent.com/15962391/162581571-149293bb-b464-4935-abcd-01511465737b.png)